pkg/kubectl/resource_printer.go

```diff
@@ -714,12 +697,12 @@ func printReplicationController(controller *api.ReplicationController, w io.Writ
 	); err != nil {
 		return err
 	}
+
+	if err := layoutContainers(containers, w, options); err != nil {
```

Comment:
I add a check inside this function, and according to our original code, we put this in wide output.